CMakeLists.txt

```diff
@@ -55,12 +55,17 @@ set(run_perf_tests ${original_run_perf_tests})
 set(run_unittests ${original_run_unittests})
 set(run_traceability ${original_run_traceability})
 
-
 set(testrunnerswitcher_c_files
-./src/testmutex.c
 ./src/ctrs_sprintf.c
```

Comment:
nitpick: can you tab this item

Reply:
Fixed here and below

---
In reply to: [562773371](https://github.com/Azure/c-testrunnerswitcher/pull/57#discussion_r562773371) [](ancestors = 562773371)